src/core_plugins/kibana/public/discover/controllers/discover.js

```diff
@@ -201,6 +201,16 @@ function discoverController(
     .highlightAll(true)
     .version(true);
 
+  // searchSource which applies time range
+  const timeRangeSearchSource = savedSearch.searchSource.new();
+  timeRangeSearchSource.set('query', null);
```

Comment:
I think we don't need this anymore, since we don't inherit the old search source anymore?